packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/generalized-react-app.js

```diff
@@ -30,7 +30,7 @@ const GROUPS = [
  * in and either 2a) append `showReactApp=true` to the URL _or_ 2b) to see it in a flow, append
  * the following query params to the page that will navigate to the page you're interested in:
  * `?forceExperiment=generalizedReactApp&forceExperimentGroup=react` */
-const ROLLOUT_RATE = 0.0;
+const ROLLOUT_RATE = 0.15;
```

Comment:
Ignorant question alert: Should this be set via an env var? I'm not sure how SRE stuff works, but if we roll out to 15% population but then need to resize that down to 0% for whatever reason, would that require a patch &raquo; dot release &raquo; redeploy? Or if this is set via an environment variable, would SRE be able to just change that env var on the server cluster to 0% without any new emergency deployments?

Reply:
Also, if this is 15%, do we need to tweak the "This experiment is disabled by default." comment on L28?

Reply:
I had suggested we maybe make it an env var but in a recent discussion, we determined it was fine to leave hard-coded. We're going to do this as a separate dot release the first time around so we can revert it if we need to. SRE can flip the feature flags on and off as they please so if there's an issue with a specific set of routes, those can be turned off.